tests/gatys_et_al_2017/test_core.py

```diff
@@ -0,0 +1,50 @@
+import pytorch_testing_utils as ptu
+
+import pystiche_papers.gatys_et_al_2017 as paper
+from pystiche_papers.gatys_et_al_2017 import utils
+
+from pystiche.image.transforms.functional._resize import resize
+
+
+def test_gatys_et_al_2017_nst_smoke(subtests, mocker, content_image, style_image):
+    mock = mocker.patch(
+        "pystiche_papers.gatys_et_al_2017.core.default_image_pyramid_optim_loop"
+    )
+
+    paper.gatys_et_al_2017_nst(content_image, style_image)
+
+    args, kwargs = mock.call_args
+    input_image, criterion, pyramid = args
+    get_optimizer = kwargs["get_optimizer"]
+    preprocessor = kwargs["preprocessor"]
+    postprocessor = kwargs["postprocessor"]
+
+    with subtests.test("input_image"):
+        ptu.assert_allclose(input_image, resize(content_image, 800))
```

Comment:
We should use `pyramid[-1].resize_image(content_image)` here. If we are sure the `pyramid` is working correctly we don't need to know the actual size of the image.